README.md

```diff
@@ -0,0 +1,155 @@
+# Terraform provider for RavenDB 
+
+## Supported Platforms
+ 
+ - Linux
+
+## Installing the provider
+
+```
+terraform {
+  required_providers {
+    ravendb = {
+      source  = "ravendb.net/ravendb/ravendb"
+      version = "1.0.0"
+    }
+  }
+}
+```
+
+## Where to Ask for Help
+
+If you have any questions, or need further assistance, you can [contact us directly](https://ravendb.net/contact).
+
+## Requirements
+
+| Name | Version |
+|------|---------|
+| terraform | >= 1.0.3 |
+
+## Providers
+
+| Name | Version |
+|------|---------|
+|ravendb|1.0.0
+
+```hcl
+terraform {
+  required_providers {
+    ravendb = {
+      source  = "ravendb.net/ravendb/ravendb"
+      version = "1.0.0"
+    }
+  }
+}
+    
+provider "aws" {
+  region = "us-east-1"
+}
+locals {
+  nodes = toset(["a", "b", "c"])
+}
+
+module "ec2_instances" {
+  source  = "terraform-aws-modules/ec2-instance/aws"
+  version = "~> 3.0"
+
+  for_each = local.nodes
+
+  name = "${each.key}.ravendb"
+
+  ami                         = "ami-09e67e426f25ce0d7"
+  instance_type               = "t2.micro"
+  key_name                    = "omer-tf"
+  monitoring                  = false
+  vpc_security_group_ids      = ["sg-03525214aee516d50"]
+  subnet_id                   = "subnet-0ee70783a7dd19aa5"
+  associate_public_ip_address = true
```

Comment:
This is probably not something that we want in the readme. Here we can just assume that the machines were provided to us